src/main/docker/Dockerfile-profiling.jvm

```diff
@@ -3,7 +3,7 @@
 # based on less minimal UBI8 OpenJDK 17 image
 #
 ###
-FROM registry.access.redhat.com/ubi8/openjdk-17:1.20
+FROM registry.access.redhat.com/ubi8/openjdk-21:1.20
```

Comment:
I assume you checked but is `1.20` the latest available minor version?

Reply:
> is 1.20 the latest available minor version?

Yes https://catalog.redhat.com/software/containers/ubi8/openjdk-21-runtime/653fd184292263c0a2f14d69?gs&q=openjdk%2021%20ubi